plugin-api/src/main/java/org/sonar/api/ce/measure/Issue.java

```diff
@@ -40,15 +41,27 @@ public interface Issue {
 
   /**
    * Available list of status can be found in {@link org.sonar.api.issue.Issue#STATUSES}
+   *
+   * @deprecated since 10.4 in favor of {@link Issue#issueStatus()}
    */
+  @Deprecated(since = "10.4")
   String status();
```

Comment:
It looks like in SonarCloud we have this class duplicated, and I don't see usages of this one. And it seems that the duplication happened before the SonarCloud fork - do you use it in SQ? Maybe it doesn't need to exist in plugin API?
@leo-geoffroy-sonarsource

Reply:
This is used by [MeasureComputer](https://github.com/SonarSource/sonar-plugin-api/blob/a0a4efc4f99dc782a7006dc3ab8eb33d87476d52/plugin-api/src/main/java/org/sonar/api/ce/measure/MeasureComputer.java), which allows a custom plugin to add computation for a new custom measure. So I guess that's why it is not used in SonarCloud, since you don't allow custom plugins.
So I would say it is still needed in the plugin-api so users can interface and create their own custom measures

Reply:
Makes sense, thanks for the clarification!

Btw we started working on the issue workflow this week, we should be able to approve/add some comments this or next week